projects/vdk-plugins/vdk-impala/tests/template_regression_test.py

```diff
@@ -0,0 +1,582 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import json
+import os
+import pathlib
+import re
+import time
+import unittest
+from unittest.mock import ANY
+from unittest.mock import MagicMock
+from unittest.mock import patch
+
+import pytest
+from vdk.internal.core import errors
+from vdk.plugin.impala import impala_plugin
+from vdk.plugin.test_utils.util_funcs import CliEntryBasedTestRunner
+from vdk.plugin.test_utils.util_funcs import get_test_job_path
+
+VDK_DB_DEFAULT_TYPE = "VDK_DB_DEFAULT_TYPE"
+VDK_IMPALA_HOST = "VDK_IMPALA_HOST"
+VDK_IMPALA_PORT = "VDK_IMPALA_PORT"
+
+
+@pytest.mark.skip(
```

Comment:
Why skipping all ~16 tests? When running tests locally, did spot 4 tests failing only currently? 

Btw 10x for the draft, will spend some time on those tests to try and resolve with present/new impala docker image

Reply:
I think that we should not be running those tests, even if they pass in the older impala version since the two versions are so different this might give some sense of false security.

Reply:
that's a good point, 
could you write a disclaimer to vdk-impala README.md, that operability is verified up to a particular impala version?